owm2ffmap/owm2ffmap.py

```diff
@@ -170,6 +177,7 @@ def process_node_json(comment, body, hostid=None, firstseen=None, lastseen=None)
         if not check_location(float(longitude), float(latitude)):
             print("...out of geographic bounds, skipping")
             return
+        # check if the router itself has an uplink. returns True or False
         isuplink = len([a for a in owmnode.get("interfaces", []) if a.get("ifname", "none") == "ffvpn"]) > 0
```

Comment:
Can you move your Hedy-uplink-test close to this line? Keeps relating code close to each other.

Reply:
thanks, looks much more nice now.